privacyidea/lib/utils/export.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+import logging
+
+log = logging.getLogger(__name__)
+
+EXPORT_FUNCTIONS = {}
+IMPORT_FUNCTIONS = {}
+
+
+def register_export_name(name):
```

Comment:
Currently it is not clear to me, why there is a `register_export_name` and a `register_export`.

I would prefer to drop the phrase "_name(name)". If we one day have exporters/importer all over the code, "name" is a bit unspecific.
Maybe the signature

~~~~python
def register_config_export(section):
~~~~

might be more intuitive, when stumbling over it somewhere in the code.

Reply:
See above, I'll use `register_export` and `register_import`.

Reply:
Done in 426389b